cmd/controller/cmd/root.go

```diff
@@ -69,8 +71,9 @@ var rootCmd = &cobra.Command{
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	version.LogBuildInformation(appName)
+	logrus.Infof("Detected: %d CPU's\n", runtime.NumCPU())
```

Comment:
why not stdlib/logger?

Reply:
I think its not initialized yet? The version code does not seem to use it?

Reply:
done!